source/appModuleHandler.py

```diff
@@ -434,6 +434,13 @@ def _get_is64BitProcess(self):
 		self.is64BitProcess = not res
 		return self.is64BitProcess
 
+	def isGoodUIAWindow(self,hwnd):
+		"""
+		returns true if the UIA implementation of the given window must be used, regardless whether native or not.

```

Comment:
Perhaps add a comment here and on `isBadUIAWindow`: "if `isGoodUIAWindow` and `isBadUIAWindow` return `False`, then the decision of whether to use UIA is left to the core."

What if they both return True?

Reply:
> What if they both return True?

isGoodUIAWindow will take precedence as UIAHandler.handler._isUIAWindowHelper will return early in that case. This is intentional, as whether good or bad, I think that the core should always have precedence over app Modules. On the other hand, appModules might want to override core behaviour for SysListView32 cases with native UIA implementations. The only case that is not covered is when the core decides good and an appModule decides bad.